pycqed/scripts/Spectroscopy.py

```diff
@@ -0,0 +1,281 @@
+"""
```

Comment:
I would suggest moving this file to `pycqed/analysis_v2/spectroscopy_analysis.py`.